tests/requirements-py3.txt

```diff
@@ -0,0 +1,4 @@
+pytest>=2.6.0
+pytest-twisted
+testfixtures
+jmespath
```

Comment:
Why is jmespath a test requirement, shouldn't it be a regular requirment? 
Sorry if you've answered me already and I forgot the answer :)